02week/ticTacToe.js

```diff
@@ -22,44 +22,81 @@ function printBoard() {
   console.log('  ---------');
   console.log('2 ' + board[2].join(' | '));
 }
-
+//code here
 function horizontalWin() {
-  // if horizontal grid has equal items, there is a win
-  
+  return board[0].every(square => square === playerTurn) || board[1].every(square => square === playerTurn) || board[2].every(square => square === playerTurn);
 }
 
 function verticalWin() {
-  // if vertical grid has equal items, there is a win
-
+  return [board[0][0], board[1][0], board[2][0]].every(square => square === playerTurn) || [board[0][1], board[1][1], board[2][1]].every(square => square === playerTurn) || [board[0][2], board[1][2], board[2][2]].every(square => square === playerTurn);
 }
 
 function diagonalWin() {
-  // if both diagonal grids have equal items, there is a win
+  return [board[0][0], board[1][1], board[2][2]].every(square => square === playerTurn) || [board[0][2], board[1][1], board[2][0]].every(square => square === playerTurn);
 }
 
 function checkForWin() {
-  //
+  if (horizontalWin()) {
```

Comment:
Consolidate your `checkForWin()` logic with `||` operators to reduce redundant code.